canonical-websites/www.ubuntu.com
# Remove finished lp AB tests

## Done

* Removing the alternate versions of `/engage/iot-business-model` and `/engage/multi-cloud-guide`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure `/engage/iot-business-model2` and `/engage/multi-cloud-guide2` have been deleted
